classes/plugins/Hook.php

```diff
@@ -66,7 +66,7 @@ public static function clear(string $hookName): void
      * @param object|array $callback Callback pseudo-type
      * @param int $hookSequence Optional hook sequence specifier SEQUENCE_...
      */
-    public static function add(string $hookName, callable $callback, int $hookSequence = self::SEQUENCE_NORMAL): void
+    public static function add(string $hookName, callable|array $callback, int $hookSequence = self::SEQUENCE_NORMAL): void
```

Comment:
Were you getting an error on a specific version of PHP? I've always understood an array `[$this, 'methodName']` to be equivalent to a callback, and I'm not sure @asmecher will want to change the hook syntax.

Reply:
PHP 8.0.28 
Sorry, error message was misleading: `Uncaught TypeError: PKP\plugins\Hook::add(): Argument #2 ($callback) must be of type callable, array given`, which appeared because of using `protected` or `private` method access modifier on a callable. I guess this is because it's not resolved within a class... Removing